include/SFML/Window/WindowBase.inl

```diff
@@ -39,6 +39,9 @@ template <typename... Ts>
 struct OverloadSet : Ts...
 {
     using Ts::operator()...;
+#if defined(_MSC_VER) && !(defined(__clang__) || defined(__INTEL_COMPILER))
```

Comment:
Is there an Intel compiler that sometimes defines `_MSC_VER`?

Reply:
https://www.intel.com/content/www/us/en/docs/dpcpp-cpp-compiler/developer-guide-reference/2023-1/additional-predefined-macros.htm